packages/grafana-data/src/themes/themeDefinitions/gildedgrove.ts

```diff
@@ -40,6 +40,7 @@ const gildedGroveTheme: NewThemeOptions = {
       canvas: '#111614',
       primary: '#1d2220',
       secondary: '#27312E',
+      elevated: '#27312E',
```

Comment:
if we want we can later tweak and use different shade for elevated, does not need to be the same as secondary. But leaving that for future iterations .